.gitignore

```diff
@@ -6,8 +6,11 @@
 *.sconsign.dblite
 *.org.eclipse.cdt.core.prefs
 *.language.settings.xml
+*.log
+*.jou
+*.project
 *.cproject
-
+WD-Firmware/demo/build/configure.txt
```

Comment:
change to configure.txt

Reply:
done and check whether the change is apropriate

Reply:
done